pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -85,7 +85,7 @@ type Cluster struct {
 	cephVersion          cephv1.CephVersionSpec
 	Count                int
 	AllowMultiplePerNode bool
-	MonCountMutex        sync.Mutex
+	monMutex             sync.Mutex
```

Comment:
Is it beneficial to call this `monMutex` over just `mutex`?

Reply:
How about `orchestrationMutex`? `mutex` would make it too easy to use for an unrelated purpose in the future.